services/settings/pkg/store/defaults/defaults.go

```diff
@@ -21,17 +21,22 @@ const (
 	// RoleManagementPermissionID is the hardcoded setting UUID for the role management permission
 	RoleManagementPermissionID string = "a53e601e-571f-4f86-8fec-d4576ef49c62"
 	// RoleManagementPermissionName is the hardcoded setting name for the role management permission
-	RoleManagementPermissionName string = "role-management"
+	RoleManagementPermissionName string = "Roles.ReadWrite"
 
 	// SettingsManagementPermissionID is the hardcoded setting UUID for the settings management permission
 	SettingsManagementPermissionID string = "3d58f441-4a05-42f8-9411-ef5874528ae1"
 	// SettingsManagementPermissionName is the hardcoded setting name for the settings management permission
-	SettingsManagementPermissionName string = "settings-management"
+	SettingsManagementPermissionName string = "Settings.ReadWrite"
 
-	// LanguageReadWriteID is the hardcoded setting UUID for the language read write permission
+	// LanguageReadWriteID is the hardcoded setting UUID for the language read write all permission
 	LanguageReadWriteID string = "7d81f103-0488-4853-bce5-98dcce36d649"
-	// LanguageReadWriteName is the hardcoded setting name for the language read write permission
-	LanguageReadWriteName string = "language-readwrite"
+	// LanguageReadWriteName is the hardcoded setting name for the language read write all permission
+	LanguageReadWriteName string = "Language.ReadWrite.All"
+
+	// LanguageReadWriteSelfID is the hardcoded setting UUID for the language read write self permission
```

Comment:
We do not need that second scope for the Language permisson. We always append the scope to the list permissions response.

## Example

```json
{
    "permissions":[
        "role-management.all",
        "settings-management.all",
        "language-readwrite.all",
        "Drive.ReadWrite.all",
        "Drive.ReadWriteEnabled.all",
        "change-logo.all",
        "create-space.all",
        "list-all-spaces.all",
        "delete-all-home-spaces.all",
        "account-management.all",
        "group-management.all",
        "delete-all-spaces.all",
        "PublicLink.Write.all",
        "set-space-quota.all",
        "Drive.ReadWriteQuota.Project.all"
    ]
}
```

Reply:
:sob: We need to change it. It makes no sense on one half of the permissions and is wrong on the other half

Reply:
Removed all constraints in names and double checked explicit constraints are correct. Please check again

Reply:
I think we are not aligned. Every permission has a scope. Example https://github.com/owncloud/ocis/blob/master/services/settings/pkg/store/defaults/defaults.go#L146